src/components/ThumbsFeedback.tsx

```diff
@@ -1,8 +1,11 @@
 import { useSearchState } from '@yext/search-headless-react';
 import { useCallback, useState } from 'react';
 import { ThumbIcon } from '../icons/ThumbIcon';
-import useLayoutEffect from 'use-isomorphic-layout-effect';
 import { useComposedCssClasses } from '../hooks/useComposedCssClasses';
+import useIsomorphicLayoutEffect from "use-isomorphic-layout-effect";
+const useLayoutEffect = typeof useIsomorphicLayoutEffect === 'function'
+  ? useIsomorphicLayoutEffect
```

Comment:
why would it be different?

Reply:
when vite is running this esm code (pagesjs dev and node), it imports the package as cjs and the export becomes an object with the "default" property, which is different than how the import works in other environments